<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPCWalkingDialog</class>
 <widget class="QDialog" name="MPCWalkingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BTN_Walk_Ready">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Walk Ready</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_FORWARD">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SAVE_START">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DATA_SAVE">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>249</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CCW">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>++CCW LEFT Rotation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CW">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>--CW Right Rotation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_LEFT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LEFT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_RIGHT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RIGHT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STEP_NUM">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STEP_LENGTH">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0.001</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STEP_ANGLE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STEP_OFFSET">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0.25</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Joy stick on</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Para Change</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_WEIGHT">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>230</string>
   </property>
  </widget>
  <widget class="QLabel" name="Weight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
